### ISSUE TITLE
Error popping up despite working fine

### ISSUE THREAD

**Issue by michaelaye** [ryantam626/jupyterlab_code_formatter#124]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**

When using the menubar icon to format the notebook, an error box pops up, even so the notebook is being formatted correctly.

This does not happen when I use Rightclick->Format Cell

**Diagnostic commands**
Please attach the output of the following commands (please format them properly)

- `pip freeze`
pip freeze output is too long.
```bash
$ pip show jupyterlab_code_formatter                                                         (py37)
Name: jupyterlab-code-formatter
Version: 1.2.1
Summary: Server extension to power `@ryantam626/jupyterlab_code_formatter` npm package.
Home-page: https://github.com/ryantam626/jupyterlab_code_formatter
Author: Ryan Tam
Author-email: ryantam626@gmail.com
License: MIT
Location: /Users/klay6683/miniconda3/envs/py37/lib/python3.7/site-packages
Requires: notebook, packaging
Required-by:
```
- `jupyter labextension list`
```bash
$ jupyter labextension list                                                                  (py37)
JupyterLab v2.0.1
Known labextensions:
   app dir: /Users/klay6683/miniconda3/envs/py37/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        @jupyterlab/geojson-extension v2.0.0  enabled  OK
        @jupyterlab/github v2.0.0  enabled  OK
        @ryantam626/jupyterlab_code_formatter v1.2.1  enabled  OK
        dask-labextension v2.0.0  enabled  OK
        ipyevents v1.7.0  enabled  OK
        ipyvolume v0.5.2  enabled  OK
        itkwidgets v0.26.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK
        jupyter-threejs v2.1.1  enabled  OK
        jupyterlab-datawidgets v6.2.0  enabled  OK
        nbdime-jupyterlab v2.0.0  enabled  OK
```
- `jupyter serverextension list`
```bash
$ jupyter serverextension list                                                               (py37)
config dir: /Users/klay6683/.jupyter
    jupyterlab_github  enabled
    - Validating...
      jupyterlab_github 1.0.0 OK
    jupyter_nbextensions_configurator  enabled
    - Validating...
      jupyter_nbextensions_configurator 0.4.1 OK
    jupyterlab_git  enabled
    - Validating...
      jupyterlab_git  OK
    nbresuse  enabled
    - Validating...
      nbresuse  OK
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /Users/klay6683/miniconda3/envs/py37/etc/jupyter
    dask_labextension  enabled
    - Validating...
      dask_labextension 2.0.0 OK
    ipyparallel.nbextension  enabled
    - Validating...
      ipyparallel.nbextension  OK
    jupyter_server_proxy  enabled
    - Validating...
      jupyter_server_proxy  OK
    jupyterlab  enabled
    - Validating...
      jupyterlab 2.0.1 OK
    jupyterlab_git  enabled
    - Validating...
      jupyterlab_git  OK
    jupyterlab_github  enabled
    - Validating...
      jupyterlab_github 1.0.0 OK
    jupyterlab_hdf  enabled
    - Validating...
      jupyterlab_hdf 0.3.0 OK
    nbdime  enabled
    - Validating...
      nbdime 2.0.0 OK
    nbresuse  enabled
    - Validating...
      nbresuse  OK
    jupyter_nbextensions_configurator  enabled
    - Validating...
      jupyter_nbextensions_configurator 0.4.1 OK
```

**Screenshots**
<img width="464" alt="Screenshot 2020-03-17 17 51 40" src="https://user-images.githubusercontent.com/69774/76912332-2045d380-6879-11ea-842d-5787b72ae764.png">

**Comment by ryantam626** [comment]
Hello :wave: 
I have already fixed the issue in 2ddd8659827bb504d2b488833d66dc72dbcd100c and released a new version (1.2.2)


`jupyter labextension install @ryantam626/jupyterlab_code_formatter@1.2.2`
`pip install jupyterlab_code_formatter==1.2.2`
to upgrade

**Comment by michaelaye** [comment]
Confirmed working! thanks much!
